solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/test/java/com/solace/spring/cloud/stream/binder/util/XMLMessageMapperTest.java

```diff
@@ -424,6 +430,9 @@ public void testMapSpringMessageToXMLMessage_OverrideDefaultSolaceProperties() t
 				case SolaceHeaders.DMQ_ELIGIBLE:
 					assertTrue(xmlMessage.isDMQEligible());
 					break;
+				case SolaceHeaders.IS_REPLY:
+					assertFalse(xmlMessage.isReplyMessage());
+					break;
```

Comment:
Remove this case. The default for `solace_isReply` was not overridden, so it doesn't apply to this test.

Reply:
done